### PR TITLE
chore(spread): miscellaneous spread fixes for core24

### PR DESCRIPTION
**Pull request by tigarmo**
- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `tox run -m lint`?
- [ ] Have you successfully run `tox run -e test-py310`? (supported versions: `py39`, `py310`, `py311`, `py312`)

-----

### CODE REVIEW

**Review by sergiusens — APPROVED** [review]
now I see where the components envvar test comes from, I made a cleanup for that, so I will consolidate after this gets in

File: tests/spread/core24/linters-file/task.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 summary: Run snapcraft lint on a snap file.
+# Disabled from regular CI runs until we have "snapcraft lint" in core24
+manual: true
```
Comment (cmatsuoka) [inline tests/spread/core24/linters-file/task.yaml:3]: We should add a note to the "snapcraft lint" task to re-enable this.

**Review by cmatsuoka — APPROVED** [review]
Nice!

File: tests/spread/core24/linters-file/task.yaml (hunk 1 above)
Comment (tigarmo) [reply tests/spread/core24/linters-file/task.yaml:3]: this is done (on the Jira issue)

**Pull request merged**